src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -823,7 +825,7 @@ public static double Log(BigInteger value, double baseValue)
             ulong l = value._bits.Length > 2 ? value._bits[value._bits.Length - 3] : 0;
 
             // Measure the exact bit count
-            int c = NumericsHelpers.CbitHighZero((uint)h);
```

Comment:
I think you can take this opportunity to kill both `CbitHighZero` overloads entirely. The only remaining call site is within `NumericsHelpers.GetDoubleFromParts`, which can be rewritten to use `BitOperations.LeadingZeroCount` as you have done here.

Reply:
Fixed